src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HPack/HPackEncoder.cs

```diff
@@ -223,6 +213,113 @@ public static bool EncodeStringLiteral(string value, Span<byte> destination, out
             return false;
         }
 
+        private static bool EncodeStringLiteralStringPart(string value, Span<byte> destination, out int bytesWritten)
```

Comment:
What does "StringPart" mean?  Can that just be "Value"?

Reply:
updated.